azext_edge/tests/edge/checks/int/test_pre_post_int.py

```diff
@@ -7,39 +7,16 @@
 import pytest
 from knack.log import get_logger
 from azure.cli.core.azclierror import CLIInternalError
-from azext_edge.edge.providers.check.common import ResourceOutputDetailLevel
-from azext_edge.edge.providers.edge_api import (
-    AkriResourceKinds,
-    DataProcessorResourceKinds,
-    LnmResourceKinds,
-    MqResourceKinds,
-    OpcuaResourceKinds,
-)
-from ...helpers import run
+from ....helpers import run
 
 logger = get_logger(__name__)
 
 
-@pytest.mark.parametrize("detail_level", ResourceOutputDetailLevel.list())
-@pytest.mark.parametrize("services_map", [
-    ("akri", AkriResourceKinds.list()),
-    ("dataprocessor", DataProcessorResourceKinds.list()),
-    ("lnm", LnmResourceKinds.list()),
-    ("mq", [
-        MqResourceKinds.BROKER.value,
-        MqResourceKinds.BROKER_LISTENER.value,
-        MqResourceKinds.DIAGNOSTIC_SERVICE.value,
-        MqResourceKinds.KAFKA_CONNECTOR.value,
-    ]),
-    ("opcua", OpcuaResourceKinds.list())
-])
+@pytest.mark.parametrize("ops_service", ["akri", "dataprocessor", "lnm", "mq", "opcua"])
```

Comment:
tbh this is not needed for testing just pre/post deployments

Reply:
Agreed, as long as the pre/post logic is completely separate from the conditional service logic - a.k.a. we don't ever add "MQ prechecks" instead of just "node prechecks"